coordinator/internal/transitengine/crypto_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package transitengine
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	r "math/rand"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/edgelesssys/contrast/internal/crypto"
+
+	"github.com/stretchr/testify/require"
+
+	b64 "encoding/base64"
+)
+
+func TestCryptoAPICyclic(t *testing.T) {
+	t.Run("encrypt-decrypt handler", func(t *testing.T) {
+		for range 10 {
```

Comment:
What is the idea of calling the test 10 times? You can also run go tests multiple times by specifying `-count 1000`. Generally, using randomness in tests is also an anti-pattern, instead create crypto related things (keys etc) beforehand and use them as constants.